sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.13.0 (Unreleased)
```

Comment:
Let's leave out the CHANGELOG entry for now since this will be in version 12.14.0 but we have to add/release 12.13.0 first :).